docs/safe-typeorm.md

```diff
@@ -0,0 +1,345 @@
+# Safe-TypeORM
```

Comment:
I think instead of having your documentation duplicated here, better just to put links to your wiki.

Reply:
To do so, I recommend to add a new section called "QueryBuilder extensions" (instead of `Safe-TypeORM`), then add a general description about QueryBuilder extension and section in there with your package name+link and description. (as the only QueryBuilder extension for now)

Reply:
Got it, I need entire re-writing.